### DISCUSSION TITLE
v0.3.1-alpha

### DISCUSSION THREAD

**Discussion by bowring** [CIRDLES/Tripoli#158] [Show and tell]
## What's Changed

Burnin on plotting is 10% of ensembles

* Bug fix and refactoring by @bowring in https://github.com/CIRDLES/Tripoli/pull/157

**Full Changelog**: https://github.com/CIRDLES/Tripoli/compare/v0.3.0-alpha...v0.3.1-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/CIRDLES/Tripoli/releases/tag/v0.3.1-alpha'>v0.3.1-alpha</a>.</em>

**Comment by noahmclean** [comment]
Requests for small UI upgrades:

- [x] When you bring up the window to choose the method file, make the window movable so that you can see the method name that is parsed from the data file.
- [x] Resize plot windows to make them bigger, and expand with frame.
- [ ] Create gesture to reject single integrations or cycles
- [ ] Zoom box

**Comment by noahmclean** [comment]
- [x] Bug: loading method multiple times duplicates species names in ratio picker (Select Ratios to plot tab)
